mmcv/ops/csrc/pytorch/cpu/sigmoid_focal_loss.cpp

```diff
@@ -0,0 +1,104 @@
+               (gamma * ((T)1. - p) * log(std::max((T)1. - p, (T)FLT_MIN)) - p);
+        }
+        if (weight != NULL) {
+            grad_input[i] *= weight[t];
+        }
+    }
+}
+
+void TensorSigmoidFocalLossForwardCPUKernelLaucher(Tensor input, Tensor target,
+                               Tensor weight, Tensor output,
+                               const float gamma,
+                               const float alpha) {
+  int output_size = output.numel();
+  int num_classes = input.size(1);
+  AT_ASSERTM(target.max().item<int64_t>() <= (int64_t)num_classes,
+             "target label should smaller or equal than num classes");
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(
+      input.scalar_type(), "sigmoid_focal_loss_forward_cpu_kernel", [&] {
+        sigmoid_focal_loss_forward_cpu_kernel(
+                output_size, input.data_ptr<scalar_t>(),
+                target.data_ptr<int64_t>(), weight.data_ptr<scalar_t>(),
+                output.data_ptr<scalar_t>(), gamma, alpha, num_classes);
+      });
+}
+
+void TensorSigmoidFocalLossBackwardCPUKernelLaucher(Tensor input, Tensor target,
+                               Tensor weight, Tensor grad_input,
+                               const float gamma,
+                               const float alpha) {
+  int output_size = grad_input.numel();
+  int num_classes = input.size(1);
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(
+      input.scalar_type(), "sigmoid_focal_loss_backward_cpu_kernel", [&] {
+        sigmoid_focal_loss_backward_cpu_kernel<scalar_t>(
+                output_size, input.data_ptr<scalar_t>(),
+                target.data_ptr<int64_t>(), weight.data_ptr<scalar_t>(),
+                grad_input.data_ptr<scalar_t>(), gamma, alpha, num_classes);
+      });
+}
+
+void sigmoid_focal_loss_forward_cpu(Tensor input, Tensor target,
+                               Tensor weight, Tensor output,
+                               const float gamma,
+                               const float alpha) {
+  TensorSigmoidFocalLossForwardCPUKernelLaucher(input, target, weight, output, gamma, alpha);
+}
+
+void sigmoid_focal_loss_backward_cpu(Tensor input, Tensor target,
+                                      Tensor weight, Tensor grad_input,
+                                      float gamma, float alpha) {
+    TensorSigmoidFocalLossBackwardCPUKernelLaucher(input, target, weight, grad_input, gamma, alpha);
+}
+void sigmoid_focal_loss_forward_impl(Tensor input, Tensor target,
+                               Tensor weight, Tensor output,
+                               const float gamma,
+                               const float alpha);
+void sigmoid_focal_loss_backward_impl(Tensor input, Tensor target,
+                                      Tensor weight, Tensor grad_input,
+                                      float gamma, float alpha);
+
+REGISTER_DEVICE_IMPL(sigmoid_focal_loss_forward_impl, CPU, sigmoid_focal_loss_forward_cpu);
```

Comment:
As the input type are the same between `TensorSigmoidFocalLossForwardCPUKernelLaucher` and `sigmoid_focal_loss_forward_cpu`, we can remote `sigmoid_focal_loss_forward_cpu` and register `TensorSigmoidFocalLossForwardCPUKernelLaucher`.
Same for `TensorSigmoidFocalLossBackwardCPUKernelLaucher`

Reply:
Thanks for your review opinion. I would fix this because it saves code.